FOS.Users.Api/FOS.Users.Api/FOS.Users.Api/Controllers/UserManagementController.cs

```diff
@@ -0,0 +1,177 @@
+﻿using AutoMapper;
+using FOS.Infrastructure.Commands;
+using FOS.Infrastructure.Queries;
+using FOS.Models.Entities;
+using FOS.Models.Requests;
+using FOS.Models.Responses;
+using MediatR;
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using static FOS.Models.Constants.Constants;
+
+namespace FOS.Users.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UserManagementController : FOSControllerBase
```

Comment:
HI @karthikrossi46 , Please rename the UserController as AuthController.
Make changes to the Front End Code to point to the AuthController endpoint..

http://localhost:<PORT>/auth/<API_NAME>